src/components/ProfileMatch/index.tsx

```diff
@@ -0,0 +1,166 @@
+            ? 'Next Court Hearing/Filing Date'
+            : 'Deadline'
+        }`
+      : `The ${
+          data.listing_type === 'CASE'
+            ? 'next court hearing/filing date'
+            : 'deadline'
+        } (${formatTimestamp(
+          data.listing_type === 'CASE' ? data.upcoming_date : data.deadline,
+        )}) is ${
+          match ? 'after' : 'before'
+        } your first available start date (${formatTimestamp(
+          profileData.start_date,
+        )}).`,
+};
+
+const matchIcon = (match: boolean | undefined) => {
+  if (match === undefined) {
+    return <Icon type="gray_dot" />; // change to yellowExclaimation
+  }
+  if (match) {
+    return <Icon type="green_check" />; // rename to greenCheck later
+  }
+  return <Icon type="redCross" />;
+};
+
+export default function ProfileMatch({
+  listingData,
+  interpretation = false,
+}: {
+  listingData: Listing;
+  interpretation?: boolean;
+}) {
+  const profile = useProfile();
+  const matchedLanguages = profile
+    ? profile.languages
+        .filter(l =>
+          listingData.listing_type === 'DOC' ? l.can_read : l.can_speak,
+        )
+        .map(l => l.language_name)
+        .filter(l => listingData.languages.includes(l))
+    : [];
+
+  const renderIconGroup = (fields: MatchField<Listing>) => {
+    if (!profile?.profileData) {
+      return null;
+    }
+    const match = fields.getMatch(listingData, profile.profileData);
+    return (
+      <Flex $gap="16px" $align="center">
+        <Flex $w="15px" $h="15px">
+          {matchIcon(match)}
+        </Flex>
+        <P>{fields.getText(listingData, profile.profileData, match)}</P>
+      </Flex>
+    );
+  };
+
+  return (
+    <Flex $direction="column" $gap="30px">
+      <H3>Profile Match</H3>
+      <Flex $direction="column" $gap="15px">
+        {listingData.listing_type === 'CASE' && (
+          <>
+            {renderIconGroup(timeCommitmentMatch as MatchField<Listing>)}
+            {renderIconGroup(locationMatch as MatchField<Listing>)}
+          </>
+        )}
+        {listingData.listing_type !== 'INT' &&
+          renderIconGroup(startDateMatch as MatchField<Listing>)}
+        <Flex $gap="16px">
+          {matchIcon(
+            listingData.listing_type === 'CASE' &&
+              !interpretation &&
+              matchedLanguages.length === 0
+              ? undefined
+              : matchedLanguages.length > 0,
+          )}
+          <P>
+            You {matchedLanguages.length > 0 ? '' : "don't "}list{' '}
+            {matchedLanguages.length > 0
+              ? parseLanguageList(matchedLanguages, false)
+              : parseLanguageList(listingData.languages, true)}{' '}
+            in your languages.
+          </P>
+        </Flex>
+      </Flex>
+      <Flex>
+        <P>
+          Go to{' '}
+          <LinkColored $color={COLORS.blueMid} href={CONFIG.settings}>
+            Profile
+          </LinkColored>{' '}
+          to change any information.
```

Comment:
Could we reword this so that it says `update` instead of `change`?

Reply:
yes!